src/pages/verify/guides/appswitch.mdx

```diff
@@ -80,4 +82,30 @@ On `iOS`, you can also use an `SFSafariViewController` if the `SFAuthenticationS
 
 </Highlight>
 
-Once the flow completes, the MitID app will perform an app-switch back to your app, which makes the `Custom Tab` / `SFAuthenticationSession` resume its operation, thus completing the login process by issuing an `OAuth2` formatted response to your native app.
\ No newline at end of file
+Once the flow completes, the MitID app will perform an app-switch back to your app, which makes the `Custom Tab` / `SFAuthenticationSession` resume its operation, thus completing the login process by issuing an `OAuth2` formatted response to your native app.
+
+
+## Web apps
+
+### Danish MitID
+
+The MitID login UX to web apps can also benefit from app-switching. 
+When using this flow, a "Start the MitID app" button will be presented to the user after they have entered their username, just as in the case for native app targets.
+Clicking this button will launch the MitID app - so the user does not have to find it and open it manually.
+
+You can activate this behavior by adding a `target:web` token in the `login_hint` query parameter.
+
+```text
+You must also add the platform-specific `appswitch:...` token as described above for native apps.
+```
```

Comment:
Intended to be a code block, or do you want to highlight this text? We use the `<Highlight>` component other places.

Reply:
(we actually use it on the same page for native app / mitid)

Reply:
> Intended to be a code block, or do you want to highlight this text? We use the `<Highlight>` component other places.

I was trying to get the `appswitch:...` to stand out, but the `<Highlight>` didn't produce that effect. Any pointers ?

Reply:
It may have been due to line breaks.

Try:

```
<Highlight icon="file-lines">

You must also add the platform-specific `appswitch:...` token as described above for native apps.

</Highlight>
```

For the contents inside the Highlight to be rendered as markdown there must be newlines after the opening tag and before the closing tag.

Reply:
> It may have been due to line breaks.
> 
> Try:
> 
> ```
> <Highlight icon="file-lines">
> 
> You must also add the platform-specific `appswitch:...` token as described above for native apps.
> 
> </Highlight>
> ```
> 
> For the contents inside the Highlight to be rendered as markdown there must be newlines after the opening tag and before the closing tag.

Ahhh, thx!

Reply:
@mickhansen Updated